app/models/project.rb

```diff
@@ -72,6 +72,18 @@ def repository_homepage
     "//#{Rails.application.config.samson.github.web_url}/#{github_repo}"
   end
 
+  def repository_homepage_gitlab
+    "//#{ENV['GITLAB_URL']}/#{github_repo}"
+  end
+
+  def github?
+    repository_url.include?('github.com')
```

Comment:
`repository_url.include? Rails.application.config.samson.github.web_url` - could make a shorter method to wrap that config variable